src/main/java/com/mentoring/ecommerce/adapter/in/web/ProductController.java

```diff
@@ -0,0 +1,25 @@
+package com.mentoring.ecommerce.adapter.in.web;
+
+import com.mentoring.ecommerce.adapter.in.web.request.ProductReq;
+import com.mentoring.ecommerce.application.port.in.SaveProductUseCase;
+import com.mentoring.ecommerce.domain.Product;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/products")
+public class ProductController {
+
+    @Autowired
+    SaveProductUseCase saveUseCase;
+
+    @PostMapping
+    public void findAllProducts(ProductReq request) {
```

Comment:
Please, change it to `final ProcutReq request`. Also, I recommend you save no characters when creating a variable or naming a class. `ProductRequest` is much better than `ProductReq`.